### DISCUSSION TITLE
Ruffle security?

### DISCUSSION THREAD

**Discussion by nagatsu-chan** [ruffle-rs/ruffle#3460] [General]
Hi ~
I was under the impression Ruffle had no security issues because it was written in Rust, but when I was scrolling through recent issues I found that there was a security issue #3444. Is this something we need to worry about while using Ruffle?

Thank u <3

**Comment by SoftCreatR** [comment]
This isn't directly related to ruffle, but with the Javascript stuff. However, this can be fixed easily, so no need to worry.
